examples/scripts/dpo.py

```diff
@@ -166,14 +167,14 @@ def process(row):
             model,
             model_ref,
             args=training_args,
-            beta=args.beta,
+            beta=training_args.beta,
```

Comment:
This arg should now live in `args` right? That is, we should remove all these params from the trainer init